hc/api/tests/test_ensuretriggers.py

```diff
@@ -13,17 +13,18 @@ def test_ensure_triggers(self):
         Command().handle()
 
         check = Check.objects.create()
-        assert check.alert_after is None
+        self.assertEqual(check.alert_after, None)
```

Comment:
This is a good start. Please use `assertIsNone`. https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNone